spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -513,13 +514,11 @@ public static class Tomcat {
 		 */
 		private DataSize maxHttpResponseHeaderSize = DataSize.ofKilobytes(8);
 
-		public DataSize getMaxHttpFormPostSize() {
-			return this.maxHttpFormPostSize;
-		}
-
-		public void setMaxHttpFormPostSize(DataSize maxHttpFormPostSize) {
-			this.maxHttpFormPostSize = maxHttpFormPostSize;
-		}
+		/**
+		 * Maximum number of parameters (GET plus POST) that will be automatically parsed
+		 * by the container.
+		 */
+		private Integer maxParameterCount;
```

Comment:
To align with [embedded Tomcat's own default](https://github.com/apache/tomcat/blob/b42637028c26a6012da1e106c3b1aeff623aeace/java/org/apache/catalina/connector/Connector.java#L213), this should be an `int` with a value of `10000`. We can then add a test to ensure that our default and Tomcat's default are in sync.